README.md

```diff
@@ -141,6 +141,24 @@ Crash address: 0x104616900
 ...
 ```
 
+### Setting up for network shaping tests (browsermobproxy)
+
+Setup
+-----
+
+1. Download the browsermob proxy zip file from http://bmp.lightbody.net/. The most current version as of this writing is browsermob-proxy-2.1.0-beta-2-bin.zip.
+2. Unpack the .zip file.
+3. Import the certificate into your Firefox profile. Select Preferences->Advanced->Certificates->View Certificates->Import... Navigate to <browsermob>/ssl-support and select cybervilliansCA.cer. Select all of the checkboxes.
```

Comment:
`<browsermob>` doesn't show up in the generated HTML. Maybe replace with `\<browsermob\>`.